api/generated.go

```diff
@@ -62,6 +69,59 @@ type registerOrganizationJSONBody Organization
 // RegisterOrganizationRequestBody defines body for RegisterOrganization for application/json ContentType.
 type RegisterOrganizationJSONRequestBody registerOrganizationJSONBody
 
+// Getter for additional properties for JWK. Returns the specified
+// element and whether it was found
+func (a JWK) Get(fieldName string) (value interface{}, found bool) {
+	if a.AdditionalProperties != nil {
+		value, found = a.AdditionalProperties[fieldName]
+	}
+	return
+}
+
+// Setter for additional properties for JWK
+func (a *JWK) Set(fieldName string, value interface{}) {
+	if a.AdditionalProperties == nil {
+		a.AdditionalProperties = make(map[string]interface{})
+	}
+	a.AdditionalProperties[fieldName] = value
+}
+
+// Override default JSON handling for JWK to handle AdditionalProperties
+func (a *JWK) UnmarshalJSON(b []byte) error {
+	object := make(map[string]json.RawMessage)
+	err := json.Unmarshal(b, &object)
+	if err != nil {
+		return err
+	}
+
+	if len(object) != 0 {
+		a.AdditionalProperties = make(map[string]interface{})
+		for fieldName, fieldBuf := range object {
+			var fieldVal interface{}
+			err := json.Unmarshal(fieldBuf, &fieldVal)
+			if err != nil {
+				return errors.Wrap(err, fmt.Sprintf("error unmarshaling field %s", fieldName))
+			}
+			a.AdditionalProperties[fieldName] = fieldVal
+		}
+	}
+	return nil
+}
+
+// Override default JSON handling for JWK to handle AdditionalProperties
+func (a JWK) MarshalJSON() ([]byte, error) {
```

Comment:
And I believe this one can be simplified to:

```
func (a JWK) MarshalJSON() ([]byte, error) {
    return json.Marshal(a.AdditionalProperties)
}
```

Reply:
generated code